rules/cross-platform/defense_evasion_deleting_websvr_access_logs.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/11/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies deletion of webserver access logs. This may indicate an attempt to evade detection or destroy forensic
+evidence on a system.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "WebServer Access Logs Deleted"
+risk_score = 21
+rule_id = "665e7a4f-c58e-4fc6-bc83-87a7572670ac"
+severity = "low"
+tags = ["Elastic", "Host", "Linux", "Windows", "Threat Detection", "Defense Evasion"]
```

macOS puts their Apache logs in same place as Linux. 
```suggestion
tags = ["Elastic", "Host", "Linux", "Windows", "macOS, "Threat Detection", "Defense Evasion"]
```